src/parser/preprocessor.c

```diff
@@ -1126,6 +1127,10 @@ static void start_function_macro_expansion(const macro_call_t *call)
 			parameter->not_specified = true;
 		}
 	}
+    if (current_call.macro->is_specially_handled){
+        //TODO: this is done temporarily for _Pragma
```

Comment:
We tend to use `/*` in cparser and always leave at least 1 space after opening the comment (similar at other places in the patch).